docs/OAUTH_PROXY.md

```
@@ -0,0 +1,328 @@
+> Bridge traditional OAuth providers to work seamlessly with MCP's authentication flow.
+
+export const VersionBadge = ({version}) => {
+  return <code className="version-badge-container">
+            <p className="version-badge">
+                <span className="version-badge-label">New in version:</span>
+                <code className="version-badge-version">{version}</code>
+            </p>
+        </code>;
+};
+
+<VersionBadge version="2.12.0" />
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: VersionBadge

(MD033, no-inline-html)